app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  validates :first_name, :last_name, :role, presence: true
+
+  enum role: [:mentor, :mentee]
```

Comment:
safer to use a hash if we're going to add to this in future:

enum :role, {mentor: 0, mentee: 1}